client/packages/invoices/src/Returns/modals/InboundReturn/InboundReturn.tsx

```diff
@@ -133,11 +143,7 @@ export const InboundReturnEditModal = ({
             ? NextStepButton
             : OkButton
         }
-        nextButton={
-          modalMode === ModalMode.Update && loadNextItem
-            ? OkAndNextButton
-            : undefined
-        }
+        nextButton={!isNewReturn ? OkAndNextButton : undefined}
```

Comment:
And this is why we need an explicit `isNewReturn` prop, as the ModalMode was no longer a correct indicator of whether or not to show the Ok/Next button